massa-execution/src/vm.rs

```diff
@@ -127,6 +127,7 @@ impl VM {
     }
 
     /// Prepare (update) the shared context before the new operation
+    /// returns a snapshot copy of the current caused ledger changes
```

Comment:
nit: If this is a separate sentence, it would be best to use an uppercase, and have a dot at the end of the line above.

Reply:
done